velox/dwio/common/Options.h

```diff
@@ -645,6 +645,7 @@ struct WriterOptions {
   std::function<std::unique_ptr<dwio::common::FlushPolicy>()>
       flushPolicyFactory;
 
+  std::string sessionTimezoneName;
```

Comment:
At this point you should use `sessionTimezone`, we don't want to look it up for every split.

Reply:
Removed 'sessionTimezone' as it is not in use. Thanks!